src/hyperlight_host/src/hypervisor/surrogate_process_manager.rs

```diff
@@ -285,12 +285,7 @@ pub(crate) fn get_surrogate_process_manager() -> Result<&'static SurrogateProces
 fn create_job_object() -> Result<HandleWrapper> {
     let security_attributes: SECURITY_ATTRIBUTES = Default::default();
 
-    let job_object = unsafe {
-        CreateJobObjectA(
-            Some(&security_attributes),
-            s!("HyperlightSurrogateJobObject"),
-        )?
-    };
+    let job_object = unsafe { CreateJobObjectA(Some(&security_attributes), PCSTR::null())? };
```

Comment:
Is there any reason to want a name?  Only thing I can think of is when debugging in ProcessExplorer

Reply:
I don't think so